README.md

```diff
@@ -93,3 +93,12 @@ To enable flushing, set `SPDLOG_FLUSH_LEVEL` to a required default (or per compo
       ```bash
       sudo /etc/init.d/ocvsmd restart SPDLOG_LEVEL=debug,ipc=off SPDLOG_FLUSH_LEVEL=warn,engine=debug
       ```
+
+### vscode
+
+Using vscode, relaunch this workspace using the appropriate devcontainer proviced. Once running in the toolshed container build, using cmake, and then do "Tasks: Run Task -> Create OCVSMD Terminals". This will launch four terminals all running in the container:
```

Comment:
```suggestion
Using vscode, relaunch this workspace using the appropriate devcontainer provided. Once running in the toolshed container build, using cmake, and then do "Tasks: Run Task -> Create OCVSMD Terminals". This will launch four terminals all running in the container:
```